### PR TITLE
Put sound on sonic KC

### PR DESCRIPTION
**Pull request by Juiceman919**

### CODE REVIEW

**Comment by JingoBurch** [comment]
I had sound for everything but the run and walk animations for sonic.

File: build/Data/CHARS/Sonic/Sonic.txt [hunk 1]
```diff
@@ -635,6 +637,7 @@ anim	run
 	frame	data/chars/Sonic/31.gif
 	offset	16 40
 	frame	data/chars/Sonic/32.gif
+	sound	data/sounds/walk.wav
```
Comment (digitalbrilliancehour) [inline build/Data/CHARS/Sonic/Sonic.txt:640]: sound fx should be on frames 3 and 6

File: build/Data/CHARS/Sonic/Sonic.txt [hunk 2]
```diff
@@ -678,6 +681,7 @@ anim	walk
 	offset	13 38
 	frame	data/chars/Sonic/16.gif
 	bbox	11 7 23 32
+	sound	data/sounds/walk.wav
```
Comment (digitalbrilliancehour) [inline build/Data/CHARS/Sonic/Sonic.txt:684]: sound fx should be on frames 4 and 6

**Review by digitalbrilliancehour — CHANGES_REQUESTED** [review]
Check out my comments on the sound fx

**Review by digitalbrilliancehour — APPROVED** [review]

**Pull request merged**